pkg/crypto/helpers_test.go

```diff
@@ -0,0 +1,35 @@
+package crypto
+
+import (
+	"testing"
+)
+
+func TestTopicFromValidCert(t *testing.T) {
+	certificate, _ := ReadPEMCertificateFile("testdata/mock_push_cert.pem")
+	pushTopic, err := TopicFromCert(certificate)
+
+	if err != nil {
+		t.Errorf("fail %s", err)
```

Comment:
You can use `t.Fatalf` here to stop the test execution if an error was returned. 
In case of `pushTopic` it doesn't matter too much because it's a string with a default value of `""`, but if you had a function which returned a pointer, the default value would be nil, likely causing a panic if the test continued. 

https://golang.org/pkg/testing/#T.Fatalf

Reply:
I agree. Fatal it is. :)